mnefun/_report.py

```diff
@@ -0,0 +1,364 @@
+                    comments = '%s["%s"] (N=%d)' % (analysis, name, evo.nave)
+                    fig = evo.plot_white(cov)
+                    report.add_figs_to_section(
+                        fig, captions=section, comments=comments,
+                        section=section, image_format='png')
+                    print('%5.1f sec' % ((time.time() - t0),))
+            else:
+                print('    %s skipped' % section)
+
+            #
+            # Source estimation
+            #
+            section = 'Source estimation'
+            if p.report_params.get('source', None) is not None:
+                t0 = time.time()
+                print(('    %s ... ' % section).ljust(ljust), end='')
+                source = p.report_params['source']
+                assert isinstance(source, dict)
+                analysis = source['analysis']
+                name = source['name']
+                # Load the inverse
+                inv_dir = op.join(p.work_dir, subj, p.inverse_dir)
+                fname_inv = op.join(inv_dir,
+                                    safe_inserter(source['inv'], subj))
+                inv = mne.minimum_norm.read_inverse_operator(fname_inv)
+                fname_evoked = op.join(inv_dir, '%s_%d%s_%s_%s-ave.fif'
+                                       % (analysis, p.lp_cut, p.inv_tag,
+                                          p.eq_tag, subj))
+                if not op.isfile(fname_inv):
+                    print('Missing inv: %s' % fname_inv)
+                elif not op.isfile(fname_evoked):
+                    print('Missing evoked: %s' % fname_evoked)
+                else:
+                    evoked = mne.read_evokeds(fname_evoked, name)
+                    title = '%s["%s"] (N=%d)' % (analysis, name, evoked.nave)
+                    stc = mne.minimum_norm.apply_inverse(
+                        evoked, inv, lambda2=source.get('lambda2', 1. / 9.),
+                        method=source.get('method', 'dSPM'),
+                        pick_ori=None)
+                    stc = abs(stc)
+                    # get clim using the reject_tmin <->reject_tmax
+                    stc_crop = stc.copy().crop(p.reject_tmin, p.reject_tmax)
+                    colormap = source.get('colormap', 'viridis')
+                    transparent = source.get('transparent', True)
+                    clim = source.get('clim', dict(kind='percent',
+                                                   lims=[82, 90, 98]))
+                    clim = mne.viz._3d._limits_to_control_points(
+                         clim, stc_crop.data, colormap)[0]
+                    clim = dict(kind='value', lims=clim)
+                    if not isinstance(stc, mne.SourceEstimate):
+                        print('Only surface source estimates currently '
+                              'supported')
+                    else:
+                        brain = stc.plot(
+                            hemi='split', views=['lat', 'med'],
```

Comment:
Do we really need the `medial` views?

![image](https://user-images.githubusercontent.com/4381290/34056228-02b4a5e2-e187-11e7-823c-fa3bd1c6cc42.png)

Reply:
depends on the paradigm. I added a commit so you can set it with `source['views']` (and you'll probably want to change `source['size']`, too), see updated example